src/api/github/getChangedStack.ts

```diff
@@ -12,14 +12,9 @@ const BACKEND_CHANGE_CHECK_NAME = 'only backend changes';
  * Given a commit and repo, check GitHub Checks to see what type of changes
  * were made in the commit (frontend, backend, fullstack)
  */
-export async function getChangedStack(
-  ref: string,
-  repo: string,
-  client?: Octokit
-) {
+export async function getChangedStack(ref: string, repo: string) {
```

Comment:
I find two calls to this function and neither uses the `client` arg. Even so, with #489 there's no material performance benefit to avoiding getClient.

Reply:
Yeah I copy pasta'd a lot of stuff from freight, so no concerns from me.